{"uuid": "a4d43d30-d322-467b-9643-a4e7e9bd2833", "children": ["85fcab95-e32b-455a-bba6-d138b4254a97"], "befores": [{"name": "get_environement_info", "status": "broken", "statusDetails": {"message": "KeyError: 'qa'\n", "trace": "  File \"c:\\users\\sf314-510g\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\users\\sf314-510g\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\users\\sf314-510g\\appdata\\local\\programs\\python\\python39\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\Users\\SF314-510G\\Desktop\\theout\\selenium-pytest-python-nonBDD\\DemoQABookStore\\testcases\\conftest.py\", line 24, in get_environement_info\n    EnvironmentInfo.get_environment_info(\"resources/test_configuration/env_conf.json\",pytest.test_env )\n  File \"C:\\Users\\SF314-510G\\Desktop\\theout\\selenium-pytest-python-nonBDD\\DemoQABookStore\\core\\env_info\\env_info.py\", line 12, in get_environment_info\n    cls.application_url = data[env]['baseUrl']\n"}, "start": 1695664660680, "stop": 1695664660680}], "start": 1695664660680, "stop": 1695664660794}